bloom-housing/bloom
# 1383/listings management, ranking and results

# Pull Request Template

## Issue

Addresses #1383 

## Description

Adds the rankings and results section to the listings management MVP. 

## Type of change


## How Can This Be Tested/Reviewed?

Create a new listing from partners, check out the new section, save the listing and checkout the detail page, re-edit the listing and ensure the data is properly loaded. You can check out Triton and Coliseum locally to see the waitlist section still rendering properly.

## Checklist:
